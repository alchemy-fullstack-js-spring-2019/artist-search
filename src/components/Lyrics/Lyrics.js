import React from 'react';
import PropTypes from 'prop-types';

function Lyrics({ lyrics }) {
  const newLines = lyrics.split('\n').map(line => (<>{ line }<br /></>));
  return (
    <p>{ newLines }</p>
  );
}

Lyrics.propTypes = {
  lyrics: PropTypes.string.isRequired
};

export default Lyrics;
