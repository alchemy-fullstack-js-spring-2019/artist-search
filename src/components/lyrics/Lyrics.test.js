import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component',  ()=>{
  it('renders stuff', ()=>{
    const wrapper = shallow(<Lyrics lyrics={'hello'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
