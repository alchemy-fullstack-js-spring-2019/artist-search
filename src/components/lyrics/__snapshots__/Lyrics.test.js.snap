// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders stuff 1`] = `
<p>
  hello
</p>
`;
